Containerfile

```diff
@@ -21,6 +21,8 @@ RUN grep -vE '^#' package-requires.txt | xargs dnf install -y && rm -f package-r
 && dnf clean all
 COPY --from=builder /build/bin/bootc-image-builder /usr/bin/bootc-image-builder
 COPY entrypoint.sh /
+COPY bib/data /usr/share/bootc-image-builder
+COPY osbuild-depsolve-dnf /yolodepsolver
```

Comment:
Yeah, there are 3 commits in the PR that needs to be removed because they embed stuff which shouldn't be embedded. Marked with the `// yolo` comment. :)